packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -321,12 +321,13 @@ export const transformPlotlyJsonToHorizontalBarWithAxisProps = (
     })
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33426&commitId=f7458fed868eaa5c120e48d1f1ef01a844dd8ad8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Gap space 25.chromium.png|2309| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Left center.chromium.png|2616| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic RTL.chromium.png|372| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>